src/rgw/rgw_file.h

```diff
@@ -802,7 +802,10 @@ namespace rgw {
 			     const uint32_t flags = RGWFileHandle::FLAG_NONE) {
       using std::get;
 
-      LookupFHResult fhr { nullptr, RGWFileHandle::FLAG_NONE };
+      // This seems like a Clang compiler error.
+      // It result in irgw::RGWFileHandle::FLAG_NONE being undefined during linking.
+      // LookupFHResult fhr { nullptr, RGWFileHandle::FLAG_NONE };
+      LookupFHResult fhr { nullptr, 0 };
```

Comment:
I'll have to produce it, which require quite a lot of recompilation. I'll post when I have it.
But it is a rather convoluted error since that same const value is referenced 2 lines earlier, and that does not result in a unknown variable during linking. So it is really fishy.

You are running Clang++-3.7? That is the Linux variant? If so in which distro?